pyproject.toml

```diff
@@ -1,12 +1,95 @@
 [build-system]
-requires = [
-    "setuptools >= 66",
-    "setuptools_scm >= 7.0.0",
-    "wheel >= 0.38.4",
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "seagap"
+dynamic = ["version"]
+description = "Community Seafloor Global Navigation Satellite Systems - Acoustic (GNSS-A) Positioning Software"
+readme = "README.md"
+license = "MIT"
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "fsspec",
+    "numba",
+    "numpy",
+    "pandas",
+    "pydantic",
+]
+
+[project.optional-dependencies]
+all = [
+    "black",
```

Comment:
Is there a way to not duplicate the dependencies here? I.e. all of these are included in the other sections (`dependencies`, `docs`, `lint`, `test`).

Reply:
Thanks for catching that! This was auto-generated from `hatch` 😅